.changeset/bright-items-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@monite/sdk-react': patch
+---
+
+feat(DEV-10563): safely take `process` if it's not available
```

Comment:
It would be better to remove the changeset because it wasn't published.

Use `yarn changeset add --empty` instead.

Reply:
Just removed the commit